pkg/osvscanner/osvscanner.go

```diff
@@ -789,6 +790,13 @@ func DoScan(actions ScannerActions, r reporter.Reporter) (models.VulnerabilityRe
 		return models.VulnerabilityResults{}, err
 	}
 
+	// ensure results are sorted for consistency
+	for _, results := range vulnsResp.Results {
```

Comment:
Can you move this into `buildVulnerabilityResults()` in vulnerability_result.go next to the code that does the results output sorting?

Reply:
Doing that at least initially doesn't give the same output - I'll do a bit more digging